string robot_name
string[] joint_names
---
bool success
string status_message                                     # comments if available
